apps/accounts/tests/test_models.py

```diff
@@ -175,3 +175,23 @@ def test_deleting_all_votes_updates_attributes_to_defaults(self):
 
         self.assertEqual(user_emot.energy, expected_new_energy)
         self.assertEqual(user_emot.valence, expected_new_valence)
+
+    def test_passing_no_update_flag_does_not_reset_boundaries(self):
```

Comment:
Remove this test, as the `update_boundaries` flag is no longer used by the delete method on UserSongVote